org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/services/XMLHighlightingTest.java

```diff
@@ -74,8 +74,8 @@ public void selfclosed() throws BadLocationException {
 
 	@Test
 	public void caseInsensivity() throws BadLocationException {

```

Comment:
I think a better name for this test case would be `caseSensitivity` or `caseNoInsensivity`

Reply:
Good catch! I rename my test with `caseSensitive` according @fbricon suggestion